tests/javascript/index.php

```diff
@@ -3897,37 +3892,10 @@ function generateAnIframeInDocument() {
 
         // Ecommerce Views
         tracker3.setEcommerceView( "SKU", "NAME HERE", "CATEGORY HERE" );
-        deepEqual( tracker3.getCustomVariable(3, "page"), ["_pks","SKU"], "Ecommerce view SKU");
-        deepEqual( tracker3.getCustomVariable(4, "page"), ["_pkn","NAME HERE"], "Ecommerce view Name");
-        deepEqual( tracker3.getCustomVariable(5, "page"), ["_pkc","CATEGORY HERE"], "Ecommerce view Category");
         tracker3.trackPageView("EcommerceView");
 
         tracker3.deleteCustomVariables('page');
 
-        // No data set
-        tracker3.setEcommerceView( );
-        deepEqual( tracker3.getCustomVariable(2, "page"), false, "No data Ecommerce price");
-        deepEqual( tracker3.getCustomVariable(3, "page"), false, "No data Ecommerce view SKU");
-        deepEqual( tracker3.getCustomVariable(4, "page"), false, "No data Ecommerce view Name");
-        deepEqual( tracker3.getCustomVariable(5, "page"), ["_pkc",""], "No data Ecommerce view Category");
```

Comment:
it's no longer possible to receive the product view values, as they are no longer stored in custom variables. Not sure if we should add a `getProductView` method in order to be possible to test that better...